views/models/UserConfigModel/index.js

```diff
@@ -83,4 +85,16 @@ const UserConfigModel = {
 export const findCategoryColor = (categoryColors, colors) =>
   colors.find(({ label }) => categoryColors.every((el, i) => el === label[i]))
 
+const processConfigAssessment = (formValues, colors) => {
+  return {
+    0: formValues.config.map(({ min, max, color, ...rest }) => {
+      return {
+        color: colors.find(({ value }) => value === color).label,
+        range: [min, max],
+        ...rest,
+      }
+    }),
+  }
+}
```

Comment:
This is the format that the folks at MGB use for configurations, I agree it could be better, like an array, and what I've learned is that there are configuration files that dpdash users upload, and they're all in this format. Ideally we would want these configurations stored clearly as an array, and to achieve that the mgb folks would have to update from the old configuration schema into a new one.